test/integration/coronavirus_local_restrictions_test.rb

```diff
@@ -148,7 +148,7 @@ class CoronavirusLocalRestrictionsTest < ActionDispatch::IntegrationTest
       then_i_enter_a_valid_english_postcode_with_a_future_level_four_restriction
       then_i_click_on_find
       then_i_see_the_results_page_for_level_three_with_changing_restriction_levels
-      then_i_do_not_see_details_of_christmas_rules
+      then_i_see_details_of_christmas_rules
```

Comment:
Are we testing to see if you're in tier 4 that you don't see Christmas rules?

Reply:
yes! https://github.com/alphagov/collections/blob/master/test/integration/coronavirus_local_restrictions_test.rb#L43

I removed it here because all we care about now that there are no tier 4 things coming in before christmas